tests/Tests/Scrapers/BullereiTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace JK\Mittagstisch\Tests;
+
+use JK\Mittagstisch\Scrapers\Bullerei;
+
+class BullereiTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var Bullerei */
+    private $instance;
+
+    public function setUp()
+    {
+        $this->instance = new Bullerei();
+    }
+
+    public function testInstance()
+    {
+        static::assertInstanceOf('JK\Mittagstisch\Scrapers\Bullerei', $this->instance);
```

Comment:
More or less point less test. You would have a point in testing it for implementing the new Restaurant interface, though.